### ISSUE TITLE
CSS and index.html issues with Markbot

### ISSUE THREAD

**Issue by riannazandbelt** [riannazandbelt/accessibility-fixer-upper#1]
Hi @mackenzierose,

I'm sorry for the long winded questions below but I'm currently stuck with this assignment as I've failed 3 accessibility tests when I run the assignment through Markbot and I've tried to make changes but nothing seems to be working. 

1. In my CSS file, Markbot says there should be a :focus state (which I have in my CSS file but maybe I wrote it incorrectly?).

2. In my index.html file, it says the following in Markbot for all of my images (grassy shrubs, shrubs with laurels and two-level shrubbery): The "Grassy shrubs" <img> tag needs more description than an alt="" can provide, aria-details="" should point to an ID with all the necessary information. I have an aria-details tag on the images, with the same corresponding id tag but once again I may have labelled them incorrectly?

3. For the remaining Markbot issues, I've attached 2 screenshots with the list of errors. For the input elements requiring labels, I've added them but Markbot doesn't seem to be recognizing them?

Any help/guidance would be greatly appreciated and apologies again for the long list of questions :)

Thanks,
Rianna

<img width="500" alt="Screen Shot 2020-04-03 at 11 58 24 AM" src="https://user-images.githubusercontent.com/55094049/78380816-7b2e3900-75a2-11ea-9b2f-98df891f4efd.png">
<img width="489" alt="Screen Shot 2020-04-03 at 11 58 37 AM" src="https://user-images.githubusercontent.com/55094049/78380819-7cf7fc80-75a2-11ea-9157-90e5ec2169ee.png">

**Comment by riannazandbelt** [comment]
@mackenzierose

**Comment by MacKenzieRose** [comment]
Hi!

The first thing I see is that you have some `<section>` tags that aren't inside of your `<main>` tags, so I would move your `<main>` to line 180 (just before your footer).

It is somewhat redundant (and I also think the source of some of the Markbot errors) to have both an `alt` and an `aria-details`; given that there is quite a lot of description for your Grassy shrubs/shrubs with laurels/two-level shrub sections, I would delete the content inside of the alt tags (so they read `alt=""`), and just leave the `aria-details` as is.

For the other recurring form errors, the issue is that the labels and inputs don't match. Ideally, we would have something like this:
`<label for="text">PUT YOUR LABEL TEXT HERE</label>`
`<input id="text" type="text" name="text">`
You'll notice I also removed the paragraph tag, and would put that content directly inside the label. (This should fix your form errors)

For your CSS file:

Because Markbot is looking for focus states on the a tags inside of the navigation, it's not seeing your styling. If you change it to read `.nav a:focus {}` that error should go away.

Start with those, and let me know how it goes.

Cheers!
MacKenzie

**Comment by riannazandbelt** [comment]
AHHH you're amazing! Thank you so much for your help, all of that worked! The last thing I'm having an issue with Markbot is the following: 

Thank you again!
Rianna 

<img width="506" alt="Screen Shot 2020-04-03 at 6 13 52 PM" src="https://user-images.githubusercontent.com/55094049/78409206-150fd900-75d7-11ea-89f8-bd5d667b1309.png">

**Comment by riannazandbelt** [comment]
@MacKenzieRose
